src/Hack.sol

```diff
@@ -752,27 +752,27 @@ contract RewardsDistributor {
 
 /// @dev This contract enables users to buy and sell tokens using the x * y = k formula,
 ///      where tokens are used to purchase tickets.
-///      The price of a ticket is the equivalent of `TICKET_PRICE_IN_ETH` Ether in token.
+///      The price of a ticket is the equivalent of `ticketPriceInEth` Ether in token.
 contract Ticketing {
     address public immutable owner;
-    uint256 public immutable TICKET_PRICE_IN_ETH;
-    uint256 public immutable VIRTUAL_RESERVE_ETH;
+    uint256 public immutable ticketPriceInEth;
+    uint256 public immutable virtualReserveEth;
     uint256 public immutable k;
 
     mapping(address => uint256) public balances;
     mapping(address => uint256) public tickets;
 
     /// @dev We assume that the values of the different parameters are big enough to minimize the impact of rounding errors.
-    /// @param ticketPriceInEth The price of a ticket in Ether.
-    /// @param virtualReserveEth The virtual reserve of Ether in the contract.
+    /// @param _ticketPriceInEth The price of a ticket in Ether.
```

Comment:
why? It's the opposite of the new pattern of the ma3 contracts lol

Reply:
idk what is the convention in this case

Reply:
Here you should find everything: 
https://www.notion.so/morpho-labs/Morpho-s-Solidity-Code-Style-a595d9ec96074bfa98ddb8017910b2c8?pvs=1&qid=1%3Afa0d54c3-37c4-46bf-805e-dbdb543feb2e%3A6

Reply:
So an underscore before the immutable variable, and no underscore before the parameter. Am I right?

Reply:
yup!